modules/S08_main_package_sbom_modules/S08_submodule_deb_package_parser.sh

```diff
@@ -0,0 +1,193 @@
+
+    for lDEB_ARCHIVE in "${lDEB_ARCHIVES_ARR[@]}" ; do
+      lR_FILE=$(file "${lDEB_ARCHIVE}")
+      if [[ ! "${lR_FILE}" == *"Debian binary package"* ]]; then
+        continue
+      fi
+
+      # if we have found multiple status files but all are the same -> we do not need to test duplicates
+      lPKG_MD5="$(md5sum "${lDEB_ARCHIVE}" | awk '{print $1}')"
+      if [[ "${lPKG_CHECKED_ARR[*]}" == *"${lPKG_MD5}"* ]]; then
+        print_output "[*] ${ORANGE}${lDEB_ARCHIVE}${NC} already analyzed" "no_log"
+        continue
+      fi
+      lPKG_CHECKED_ARR+=( "${lPKG_MD5}" )
+
+      mkdir "${TMP_DIR}/deb_package/"
+      ar x "${lDEB_ARCHIVE}" --output "${TMP_DIR}/deb_package/"
+
+      if [[ ! -f "${TMP_DIR}/deb_package/control.tar.xz" ]]; then
+        write_log "[-] No debian control.tar.xz found for ${lDEB_ARCHIVE}" "${LOG_PATH_MODULE}/${lPACKAGING_SYSTEM}.txt"
+      fi
+
+      tar xf "${TMP_DIR}/deb_package/control.tar.xz" -C "${TMP_DIR}/deb_package/"
+
+      if [[ ! -f "${TMP_DIR}/deb_package/control" ]]; then
+        write_log "[-] No debian control extracted for ${lDEB_ARCHIVE}" "${LOG_PATH_MODULE}/${lPACKAGING_SYSTEM}.txt"
+      fi
+
+      lAPP_NAME=$(grep "Package: " "${TMP_DIR}/deb_package/control" || true)
+      lAPP_NAME=${lAPP_NAME/*:\ }
+      lAPP_NAME=$(clean_package_details "${lAPP_NAME}")
+
+      lAPP_ARCH=$(grep "Architecture: " "${TMP_DIR}/deb_package/control" || true)
+      lAPP_ARCH=${lAPP_ARCH/*:\ }
+      lAPP_ARCH=$(clean_package_details "${lAPP_ARCH}")
+
+      lAPP_MAINT=$(grep "Maintainer: " "${TMP_DIR}/deb_package/control" || true)
+      lAPP_MAINT=${lAPP_MAINT/*:\ }
+      lAPP_MAINT=$(clean_package_details "${lAPP_MAINT}")
+
+      lAPP_DESC=$(grep "Description: " "${TMP_DIR}/deb_package/control" || true)
+      lAPP_DESC=${lAPP_DESC/*:\ }
+      lAPP_DESC=$(clean_package_details "${lAPP_DESC}")
+
+      lAPP_LIC=$(grep "License: " "${TMP_DIR}/deb_package/control" || true)
+      lAPP_LIC=${lAPP_LIC/*:\ }
+      lAPP_LIC=$(clean_package_details "${lAPP_LIC}")
+
+      lAPP_VERS=$(grep "Version: " "${TMP_DIR}/deb_package/control" || true)
+      lAPP_VERS=${lAPP_VERS/*:\ }
+      lAPP_VERS=$(clean_package_details "${lAPP_VERS}")
+      lAPP_VERS=$(clean_package_versions "${lAPP_VERS}")
+
+      lMD5_CHECKSUM="$(md5sum "${lDEB_ARCHIVE}" | awk '{print $1}')"
+      lSHA256_CHECKSUM="$(sha256sum "${lDEB_ARCHIVE}" | awk '{print $1}')"
+      lSHA512_CHECKSUM="$(sha512sum "${lDEB_ARCHIVE}" | awk '{print $1}')"
+
+      lAPP_VENDOR="${lAPP_NAME}"
+      lCPE_IDENTIFIER="cpe:${CPE_VERSION}:a:${lAPP_VENDOR}:${lAPP_NAME}:${lAPP_VERS}:*:*:*:*:*:*"
+
+      if [[ -z "${lOS_IDENTIFIED}" ]]; then
+        lOS_IDENTIFIED="debian-based"
+      fi
+      lPURL_IDENTIFIER=$(build_purl_identifier "${lOS_IDENTIFIED:-NA}" "deb" "${lAPP_NAME:-NA}" "${lAPP_VERS:-NA}" "${lAPP_ARCH:-NA}")
+      local lSTRIPPED_VERSION="::${lAPP_NAME}:${lAPP_VERS:-NA}"
+
+      if command -v jo >/dev/null; then
```

Comment:
That not a dependency?

Reply:
now it is ... I will remove this check soon